notebooks/book1/02/gamma_dist_plot.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        for a, b, p in zip(a_list, b_list, props):
</pre><p><code>for a, b, prob </code>  would be better?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done